docs/standard-library/forward-list-class.md

```diff
@@ -345,16 +345,16 @@ template <class Type>
 
 ### Parameters
 
-*val*\
+*`val`*\
 The element added to the beginning of the forward list.
 
 ### Remarks
 
-This member function inserts an element with the constructor arguments `_ val` at the end of the controlled sequence.
+This member function inserts an element with the constructor arguments `val` at the beginning of the controlled sequence.
```

Comment:
There is no such argument as `_ val` and `emplace_front` inserts to the beginning and not end.

Reply:
Should I laugh or cry here? Good thing its Friday.